Automattic/woocommerce-payments
# In-Person Payments: represent transactions as card payments.

Fixes https://github.com/Automattic/woocommerce-payments/issues/4101

#### Changes proposed in this Pull Request

- Fixes payment method display for In-Person Payments

#### Testing instructions

* See the server PR available via ticket
* Make In-Person payment (regular or Interac)
* Ensure client lists the payment as card apyment

-------------------


**Post merge**
